zig/leveldb.zig

```diff
@@ -89,6 +91,85 @@ pub export fn leveldb_db_get(db_ptr: u64, key: [*c]const u8, key_len: u32) u64 {
     return @intFromPtr(value.ptr);
 }
 
+// Single value representation when returning many values
+pub const ResultRef = extern struct {
+    // Pointer to the value allocated with malloc within LevelDB
+    ptr: u64,
+    // Length of the item returned from the leveldb
+    len: u32,
+    // Define if the result is empty or does not exists
+    // To cross platform C ABI compatibility we are using u8 instead of bool
+    found: u8,
+};
```

Comment:
```suggestion
/// Single value representation when returning many values
pub const ResultRef = extern struct {
    /// Pointer to the value allocated with malloc within LevelDB
    ptr: u64,
    /// Length of the item returned from the leveldb
    len: u32,
    /// Define if the result is empty or does not exists
    // To cross platform C ABI compatibility we are using u8 instead of bool
    found: u8,
};
```

Make these doc comments if need be, but I found these comments to be rather redundant other than the comment on `ptr`. For example it seems unambiguous that `len` is the 'Length of the item returned from the leveldb'